doc/nimdoc.css

```diff
@@ -78,63 +78,45 @@ Modified by Boyd Greenfield and narimiran

```

Comment:
Why is an embedded image being used here?

Reply:
I just copied the `[data-theme="dark"]` rule-set, not overwriting it means the light version of the image would be used.